liferay-echo/liferay-portal
# LPS-202230 Change Configuration view for Message Boards

## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPS-202230)

## Changes

Change the configuration for Message Boards because it should be open in a new page instead of a modal.

## Test Scenario 1

1. Go to Message Boards
2. Click on the kebab menu in the top right
3. Navigate to the sections to check the behaviour and styles

![image](https://github.com/liferay-echo/liferay-portal/assets/93203423/af328408-9ecf-4992-bc5d-2c779f80bae7)
